assets/app/scripts/services/data.js

```diff
@@ -704,7 +704,10 @@ angular.module('openshiftConsole')
   DataService.prototype._uniqueKeyForResourceContext = function(resource, context) {
     // Note: when we start handling selecting multiple projects this
     // will change to include all relevant scope
-    if (context.project && context.project.metadata) {
+    if (resource === "projects") { // when we are loading the projects list we dont need additional context
```

Comment:
Are there other resources that don't need a namespace? `projectrequests`?

/cc @liggitt